types/submenu.d.ts

```diff
@@ -8,6 +8,18 @@ export declare class ElSubmenu extends ElementUIComponent {
   /** Delay time before show a sub-menu */
   showTimeout: number
 
-  /** Delay time before hide a sub-menu */
+  /** Delay time before showing a sub-menu */
+  showTimeout: number
```

Comment:
this is duplicate, declaration.
caused module to throw error when building with webpack.
Can you please remove the duplicate declaration of type for - showTimeout form type file.

Reply:
It's already removed.